### ISSUE TITLE
Not functional with versions above 0.1-Old

### ISSUE THREAD

**Issue by nd10s** [thundersen/timberborn_vevantzedata#3]
Hey - 

This mod works with old versions of Timberborn, seemingly before the storage overhaul, but not with the newest releases. 

Error looks like this:
`[Info   :   BepInEx] Loading [Ve Vant Ze Data Timberborn! 0.1.5]
[Warning:  HarmonyX] AccessTools.DeclaredMethod: Could not find method for type Timberborn.Goods.GoodService and name Initialize and parameters
[Error  : Unity Log] ArgumentException: Undefined target method for patch method static void VeVantZeData.Collector.CaptureGameInstancesPatches+GoodServiceInitialize::Postfix(Timberborn.Goods.GoodService __instance)
Stack trace:
HarmonyLib.PatchClassProcessor.PatchWithAttributes (System.Reflection.MethodBase& lastOriginal) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
Rethrow as HarmonyException: Patching exception in method null
HarmonyLib.PatchClassProcessor.ReportException (System.Exception exception, System.Reflection.MethodBase original) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.PatchClassProcessor.Patch () (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.Harmony.<PatchAll>b__11_0 (System.Type type) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.Harmony.PatchAll (System.Reflection.Assembly assembly) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
HarmonyLib.Harmony.CreateAndPatchAll (System.Reflection.Assembly assembly, System.String harmonyInstanceId) (at <474744d65d8e460fa08cd5fd82b5d65f>:0)
VeVantZeData.Collector.Plugin.Awake () (at <a8521e18b33a47699982250423f7fefe>:0)
UnityEngine.GameObject:AddComponent(Type)
BepInEx.Bootstrap.Chainloader:Start()
UnityEngine.InputSystem.InputSystem:.cctor()`

Are there any plans to update the mod?

If not, could you let me know what I need to do to update it?

Awesome mod! Thanks!

**Comment by kringalf** [comment]
yeah, i have the same issue and would love to use this mod with current timberborn versions
